boost-maven/boost-runtimes/runtime-openliberty/src/main/java/boost/runtimes/LibertyRuntime.java

```diff
@@ -121,183 +115,34 @@ public void doPackage() throws BoostException {
     private void packageLiberty(List<AbstractBoosterConfig> boosterConfigs) throws MojoExecutionException {
         createLibertyServer();
 
-        /**
-         * Whether the packaged Liberty Uber JAR will be the project artifact. This
-         * should be the case in Spring Boot scenarios since Spring Boot developers
-         * expect a runnable JAR.
-         */
-        boolean attach;
+         // Dealing with an EE based app
```

Comment:
We can probably get rid of this comment since we are only handling EE based apps now